src/workers/cdc.ts

```diff
@@ -101,6 +103,12 @@ const isFreeformPostChangeLongEnough = (
       freeform.payload.after.content.length,
   ) >= FREEFORM_POST_MINIMUM_CHANGE_LENGTH;
 
+const isCollectionUpdated = (
```

Comment:
I see we described this in the DR indeed just a question for @idoshamun did we not want this notification to trigger when new posts got added rather?
(I think this was discussed, but rather ask to verify)

Reply:
Eventually, we decided content, if not mistaken to throttle the amount of notifications. But I'm not strongly opinionated here

Reply:
Yes, no point in notifying users if content itself did not change.

Reply:
Was more from a user point, you might want to know the new posts for the collection rather then a summary that changed.
But we can go with this for now and adapt later.

Reply:
So nothing from my side to change here?

Reply:
All good for now yes.